src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -483,7 +486,8 @@ def _search_role_assignments(cli_ctx, assignments_client, definitions_client,
             role_id = _resolve_role_id(role, scope, definitions_client)
             assignments = [i for i in assignments if worker.get_role_property(i, 'role_definition_id') == role_id]
 
-        if assignee_object_id:
+        # filter the assignee if "include_groups" is not provided
+        if assignee_object_id and not include_groups:
```

Comment:
Is this still necessary? I think `f = f + " and assignedTo('{}')".format(assignee_object_id)` already has the filter.